webplatformcompat/migrations/0022_sections_name_allow_blank.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
For Django-generated migrations, I add a line to avoid flake8 checks:

```
# flake8: noqa
```
See [an existing migration](https://github.com/mdn/browsercompat/blob/master/webplatformcompat/migrations/0001_initial.py#L2) for an example with context.

For data migrations, I go ahead and adjust whitespace so that ``flake8`` or ``make qa`` passes.